tests/snappy_pipeline/workflows/conftest.py

```diff
@@ -16,6 +17,13 @@
 from snappy_pipeline.workflows.abstract import BaseStep
 
 
+@pytest.fixture(autouse=True)
```

Comment:
@eudesbarbosa I believe this was missing. I don't understand why the tests ran through otherwise, but at least when failing for another reason, subsequent tests apparently were missing this. Some wgs_sv_export tests failed for some reason and then also there were failures regarding the partition: medium entries in the dicts.

Reply:
We probably just struck luck with the other of tests, `snappy-snake` is one of the first ones...